user-onboarding/src/main/java/br/com/edsilfer/android/user_onboarding/presenter/ActivityUserOnboarding.kt

```diff
@@ -81,7 +82,7 @@ open class ActivityUserOnBoarding : AppCompatActivity(), OnBoardingBottomPanel.P
             override fun onPageSelected(position: Int) {
                 mCurrentPage = position
                 mPanelControl!!.updateBar(mCurrentPage)
-                mViewPager!!.setBackgroundColor(mTheme!!.pages[position].background)
+                mViewPager!!.setBackgroundColor(ContextCompat.getColor(baseContext, mTheme!!.pages[position].background))
             }
```

Comment:
There was a bug which caused background to blink. You used color resource id as a color value here.